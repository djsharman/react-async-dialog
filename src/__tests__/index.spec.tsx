import DialogProvider, { useDialog } from ".."

describe("react-async-dialog/index", () => {
  describe("DialogProvider", () => {
    it.skip("renders", () => {})

    it.skip("mounts modal to non-document.body element", () => {})

    it.skip("allow override default layout", () => {})
  })

  describe("useDialog", () => {
    it.skip("subscribes parent context correctly", () => {})
  })
})
