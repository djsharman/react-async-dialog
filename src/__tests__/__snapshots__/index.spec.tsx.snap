// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`react-async-dialog/DialogProvider just has children opens modal 1`] = `"<div><button id=\\"push-me\\">push me</button><div>This message should be shown below the button</div></div><div></div><div data-dialog-root=\\"true\\" style=\\"position: fixed; background-color: rgba(24, 24, 36, 0.7); left: 0px; top: 0px; width: 100%; height: 100%; display: flex; justify-content: center; align-items: center;\\"><div style=\\"width: 600px; max-width: 100%; background: rgb(255, 255, 255); border-radius: 8px;\\"><div style=\\"padding: 16px;\\">Are you <strong>REALLY</strong> sure?</div><div style=\\"display: flex; justify-content: flex-end; padding: 8px 8px 8px 12px; border-top: 1px solid #ddd;\\"><button style=\\"appearance: none; cursor: pointer; border: 0px; border-radius: 4px; margin: 0px 4px; font-size: 0.875em; background-color: rgb(68, 102, 238); color: rgb(255, 255, 255); display: flex; justify-content: flex-end; padding: 8px 16px; border-top: 1px solid #ddd;\\">YES</button><button style=\\"appearance: none; cursor: pointer; border: 0px; border-radius: 4px; margin: 0px 4px; font-size: 0.875em; background-color: rgb(196, 196, 196); color: rgb(255, 255, 255); display: flex; justify-content: flex-end; padding: 8px 16px; border-top: 1px solid #ddd;\\">NO</button></div></div></div>"`;

exports[`react-async-dialog/DialogProvider just has children renders 1`] = `"<div>Hello World</div>"`;
